common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -86,9 +86,9 @@ class CourseFactory(XModuleFactory):
     """
     Factory for XModule courses.
     """
-    org = factory.Sequence(lambda n: 'org.%d' % n)
-    number = factory.Sequence(lambda n: 'course_%d' % n)
-    display_name = factory.Sequence(lambda n: 'Run %d' % n)
+    org = Sequence(lambda n: 'org.%d' % n)
+    number = Sequence(lambda n: 'course_%d' % n)
+    display_name = Sequence(lambda n: 'Run %d' % n)
```

Comment:
Optional: while you're editing these lines, maybe just switch them over to `'org.{}'.format` like the lines in `LibraryFactory`.